internal/services/send_receiver_wallets_invite_service.go

```diff
@@ -207,6 +202,28 @@ func (s SendReceiverWalletInviteService) SendInvite(ctx context.Context, receive
 	})
 }
 
+// resolveReceiverWalletsPendingRegistration returns the receiver wallets pending registration based on the receiverWalletInvitationData.
```

Comment:
It would be great to add what happens when the `receiverWalletInvitationData` is empty. Like adding it'll query all receivers that are pending registration from the database.

Reply:
Addressed in c4ca6f0bf17d1f1a0b0f5d3a73bf967f3950fa99